cromwell-drs-localizer/src/main/scala/drs/localizer/DrsLocalizerMain.scala

```diff
@@ -94,6 +90,8 @@ object DrsLocalizerMain extends IOApp {
     val requestBody = raw"""{"url":"$drsUrl"}"""
     val marthaErrorMsg = s"Something went wrong while trying to resolve $drsUrl through Martha $marthaUrl."
 
+    logger.info("Resolving DRS uri through Martha")
```

Comment:
Is the uri logged elsewhere? If not, can this message print out what it's resolving?

Reply:
Inside the logs, it does print what the script is localizing. So before DRS localization starts, it prints
```
Localizing input drs://abc/123 -> /cromwell_root/...
```
But I can add the uri in the log message on line 93. There is no harm in having more information